### ISSUE TITLE
Movie list does not load when logged in as user with a maximum allowed parental rating and viewing a combined library folder

### ISSUE THREAD

**Issue by SteveDinn** [jellyfin/jellyfin#5939]
**Describe the bug**

I've been as accurate as I could be in the title of this issue. When the following conditions are met, movies do not populate the list in which they are supposed to show up:

- The current user is not an admin.
- The current user has a "maximum allowed parental rating" set by an admin
- The current user has two or more libraries combined for viewing
- The combined library is attempted to be viewed.

From the logs, this appears to be a permissions issue, perhaps due to the "maximum allowed parental rating". However, if the libraries are not combined, the same user can view all of the individual libraries and just see the movies they are allowed to see.

**System (please complete the following information):**
 - OS: Ubuntu 20.04
 - Virtualization: Docker
 - Clients: Windows
 - Browser: Chrome
 - Jellyfin Version: 10.7.2
 - Playback: n/a
 - Installed Plugins: Trakt, Playback Reporting, but I don't believe these apply here.
 - Reverse Proxy: Caddy, but really this doesn't apply either.
 - Base URL: none
 - Networking: Host
 - Storage: Local

**To Reproduce**

1. Have multiple libraries containing movies.
2. Create a new user (non-admin).
3. In the admin dashboard for that user, on the Parental Control tab, set a maximum-allowed parental rating, eg. CA-PG.
4. Log in as the new user.
5. In the user's settings, under "Home" in the "Library Folders" section, check at least two libraries containing movies.
6. Go back to the home screen for the user, and attempt to navigate into the combined "Movies" library.
7. The spinner gif spins forever, and no movies ever show.

**Expected behavior**

The list of movies in the combined library should show, filtered by the user's maximum-allowed parental rating.

**Logs**

```
Jellyfin.Api.Controllers.ItemsController: "MyKidsName" is not permitted to access Library "Movies"
```
In this log entry, "Movies" is the name of the combined library folder.

**Screenshots**

![image](https://user-images.githubusercontent.com/3911507/116734494-b2e4fb80-a9c3-11eb-90d7-176de40afe1d.png)
![image](https://user-images.githubusercontent.com/3911507/116734562-c85a2580-a9c3-11eb-9ed8-9a317f6f7181.png)
![image](https://user-images.githubusercontent.com/3911507/116734602-d7d96e80-a9c3-11eb-82cc-ea816d15df95.png)

**Additional context**

**Comment by SteveDinn** [comment]
I realize that no activity has occurred on this bug, but I wanted to confirm that it is still happening in 10.7.7.

**Comment by LubricantJam** [comment]
This is still a persistent issue, not being able to access Playlists without direct linking someone to their playlist.

**Comment by shukr** [comment]
This is still an issue on 10.8.7. When user's access "Enable access to all libraries" is unchecked: 
![image](https://user-images.githubusercontent.com/1555139/203640668-86a5d145-faf3-431e-a0bd-65bc143c0a84.png)

And when user logs in and configures "Automatically group content from the following folders into views such as 'Movies', 'Music' and 'TV':"
![image](https://user-images.githubusercontent.com/1555139/203641043-917556b1-8926-470f-9573-9b9dd27dc64a.png)
![image](https://user-images.githubusercontent.com/1555139/203641093-74a9b92a-4d22-4098-99f6-16953e27b155.png)
![image](https://user-images.githubusercontent.com/1555139/203641141-64a2991a-218b-4008-b7b4-2fbff42426b0.png)

Then newly generated folder view fails to load indefinitely:
![image](https://user-images.githubusercontent.com/1555139/203641334-838b33e9-f650-40b5-8eca-c1d5aa630500.png)

**Comment by Shadowghost** [comment]
From a quick test this should be fixed by #8526
